config.js

```diff
@@ -94,7 +94,12 @@ config.mqtt = {
     /**
      * Time to keep connection open between client and MQTT broker (default is 0 seconds)
      */
-    keepalive: 0
```

Comment:
Please add an entry in CHANGES_NEXT_RELEASE about the modifications in this PR. Suggestion:

```
- Add: config.mqtt.avoidLeadingSlashes flag (IOTA_MQTT_AVOID_LEADING_SLASHES) to avoid leading slashes in MQTT
```

or (depending on the name of the parameter at the end):

```
- Add: config.mqtt.avoidLeadingSlash flag (IOTA_MQTT_AVOID_LEADING_SLASH) to avoid leading slash in MQTT
```